src/NuGet.Services.AzureSearch/VersionList/MutableIndexChanges.cs

```diff
@@ -190,6 +190,24 @@ private void MergeSearchIndexChanges(SearchFilters searchFilters, SICT addedType
             }
         }
 
+        public IndexChanges Solidify()
+        {
+            // Verify that the running list of hijack changes is the same as the pre-computed hijack document.
+            Guard.Assert(HijackDocuments.Count == HijackDocuments.Count, "The hijack document state has diverged.");
```

Comment:
>HijackDocuments.Count [](start = 50, length = 21)

Shoudl this be looking at `HijackChanges`?

Reply:
Nice catch!